ui-components/src/helpers/tableSummaries.tsx

```diff
@@ -1,81 +1,135 @@
 import * as React from "react"
 import { t } from "./translator"
-import { UnitSummary } from "@bloom-housing/backend-core/types"
+import { UnitSummary, UnitsSummary } from "@bloom-housing/backend-core/types"
 import { GroupedTableGroup } from "../tables/GroupedTable"
 
-export const unitSummariesTable = (summaries: UnitSummary[]) => {
-  const unitSummaries = summaries.map((unitSummary) => {
-    const minIncome =
-      unitSummary.minIncomeRange.min == unitSummary.minIncomeRange.max ? (
-        <strong>{unitSummary.minIncomeRange.min}</strong>
-      ) : (
-        <>
-          <strong>{unitSummary.minIncomeRange.min}</strong> {t("t.to")}{" "}
-          <strong>{unitSummary.minIncomeRange.max}</strong>
-        </>
-      )
+export const getSummaryRow = (
+  minIncomeRangeMin?: string,
+  minIncomeRangeMax?: string,
+  rentRangeMin?: string,
+  rentRangeMax?: string,
+  rentAsPercentIncomeRangeMin?: string,
+  rentAsPercentIncomeRangeMax?: string,
+  totalAvailable?: number,
+  totalCount?: number,
+  unitTypeName?: string
+) => {
+  let minIncome = <></>
+  if (minIncomeRangeMin == undefined) {
+    //TODO(#345): figure out what to display when there's no data
+    minIncome = <strong>{t("t.call")}</strong>
+  } else if (minIncomeRangeMin == minIncomeRangeMax) {
+    minIncome = (
+      <strong>
+        {minIncomeRangeMin}
+        {t("t.perMonth")}
```

Comment:
nit: do we want "per month" here? It wasn't there in the original version of this file, and I'm not sure if the income we're storing is a yearly or a monthly income.

Same thing on line 33.

Reply:
This was originally in the return statement on line 47 of Bloom's version.  This is just where we pulled it up.  I think we need some type of designator for people to understand.  So be it "per month" or "per year", we should have something.

Reply:
Ah gotcha, sorry for missing that! Sounds good on pulling it up.

One little thing: can you take it out from within `<string>...</strong>`? I'm imagining:

```
      <strong>
        {minIncomeRangeMin}
      </strong>
      {t("t.perMonth")}
```

(same below, where "per month" is inside the `strong` tags too)

I think that will make it mimic what Exygy had before (and will be something they'll ask for in their review), where the "per month" isn't bolded.

Reply:
Updated